paper_eval.py

```diff
@@ -77,79 +103,102 @@
 logger.info("Using output dir: %s" % model_logging_dir)
 
 parquet_file: str = args["parquet_file"]
-val_samples_file: str = args["val_samples_file"]
+# val_samples_file: str = args["val_samples_file"]
 train_only_samples_file: str = args["train_only_samples_file"]
 
 path_to_config = config_dir / "default.json"
 model_kwargs = json.load(Path(path_to_config).open("r"))
 
-logger.info("Setting up dataloaders")
+model_modes = ["CatBoostClassifier"]
+# model_modes = ["Random Forest", "Regression", "CatBoostClassifier"] 
 
+logger.info("Loading data")
 df = pd.read_parquet(data_dir / parquet_file)
 
-logger.info("Setting up model")
-if warm_start:
-    model_kwargs = json.load(Path(config_dir / "default.json").open("r"))
-    model = Presto.load_pretrained()
-    best_model_path: Optional[Path] = default_model_path
-else:
-    if path_to_config == "":
-        path_to_config = config_dir / "default.json"
-    model_kwargs = json.load(Path(path_to_config).open("r"))
-    model = Presto.construct(**model_kwargs)
-    best_model_path = None
-model.to(device)
+val_samples_file = f"{task_type}_{test_type}_generalization_test_split_samples.csv"
```

Comment:
This approach requires the script to be run 3 times to get the full results (instead of only once to get all 3 results, which was the case previously). 

Perhaps we can update this so that a single run gets all the data?